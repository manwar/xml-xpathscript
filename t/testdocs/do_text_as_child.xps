<%
    $template->set( 'text()' => { pre => 'bar' } );
%><%= apply_templates() %>
<%
    $template->set( 'text()' => { action => DO_TEXT_AS_CHILD, 
                                  pre    => 'text: ',             } );
%><%= apply_templates() %>
