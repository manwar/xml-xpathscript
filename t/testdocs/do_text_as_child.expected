<doc>bar</doc>
<doc>text: foo</doc>
